From a7dc14c5298458cce56b4ad9b3d92b79435f24de Mon Sep 17 00:00:00 2001
From: Pratyush Yadav <p.yadav@ti.com>
Date: Fri, 21 Apr 2023 11:33:42 +0530
Subject: [PATCH 315/508] media: ti: j721e-csi2rx: rename csi->subdev to
 csi->source

A separate subdev will be added for the main TI CSI2RX device which will
take care of the routing. Rename the subdev to source so that the
difference between the two will be clear. It is a better name anyway
since it more accurately reflects the purpose of the member.

Signed-off-by: Pratyush Yadav <p.yadav@ti.com>
Signed-off-by: Jai Luthra <j-luthra@ti.com>
---
 drivers/media/platform/ti/j721e-csi2rx/j721e-csi2rx.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/media/platform/ti/j721e-csi2rx/j721e-csi2rx.c b/drivers/media/platform/ti/j721e-csi2rx/j721e-csi2rx.c
index 1682f8378353..df321fa882c2 100644
--- a/drivers/media/platform/ti/j721e-csi2rx/j721e-csi2rx.c
+++ b/drivers/media/platform/ti/j721e-csi2rx/j721e-csi2rx.c
@@ -104,7 +104,7 @@ struct ti_csi2rx_dev {
 	struct media_pipeline		pipe;
 	struct media_pad		pad;
 	struct v4l2_device		v4l2_dev;
-	struct v4l2_subdev		*subdev;
+	struct v4l2_subdev		*source;
 	struct ti_csi2rx_ctx		ctx[TI_CSI2RX_NUM_CTX];
 };
 
@@ -335,7 +335,7 @@ static inline int ti_csi2rx_video_register(struct ti_csi2rx_ctx *ctx)
 	if (ret)
 		return ret;
 
-	ret = v4l2_create_fwnode_links_to_pad(csi->subdev, &csi->pad,
+	ret = v4l2_create_fwnode_links_to_pad(csi->source, &csi->pad,
 					      MEDIA_LNK_FL_IMMUTABLE | MEDIA_LNK_FL_ENABLED);
 
 	if (ret) {
@@ -352,7 +352,7 @@ static int csi_async_notifier_bound(struct v4l2_async_notifier *notifier,
 {
 	struct ti_csi2rx_dev *csi = dev_get_drvdata(notifier->v4l2_dev->dev);
 
-	csi->subdev = subdev;
+	csi->source = subdev;
 
 	return 0;
 }
@@ -653,7 +653,7 @@ static int ti_csi2rx_start_streaming(struct vb2_queue *vq, unsigned int count)
 
 	spin_unlock_irqrestore(&dma->lock, flags);
 
-	ret = v4l2_subdev_call(csi->subdev, video, s_stream, 1);
+	ret = v4l2_subdev_call(csi->source, video, s_stream, 1);
 	if (ret)
 		goto err_dma;
 
@@ -677,7 +677,7 @@ static void ti_csi2rx_stop_streaming(struct vb2_queue *vq)
 
 	video_device_pipeline_stop(&ctx->vdev);
 
-	ret = v4l2_subdev_call(csi->subdev, video, s_stream, 0);
+	ret = v4l2_subdev_call(csi->source, video, s_stream, 0);
 	if (ret)
 		dev_err(csi->dev, "Failed to stop subdev stream\n");
 
-- 
2.41.0

